lib/scope.js

```diff
@@ -420,13 +459,23 @@ module.exports = {
     await scope.tapElement( scope, elem );
   },
 
+  tapTab: async function ( scope, tab_id ) {
+    let elem = await scope.page.waitForSelector(`#${tab_id}`);
+    await scope.tapElement( scope, elem, tabToWaitFor=tab_id );
+  },
+
+  tapElementBySelector: async function ( scope, selector, waitForTimeout ) {
+    let elem = await scope.page.waitForSelector(`${selector}`);
+    await scope.tapElement (scope, elem, false, waitForTimeout );
+  },
+
   load: async function ( scope, file_name ) {
     /* Try to load the page. Should we also pass a `timeout` arg or just
     *    use scope.timeout? */
 
     // If there is no browser open, start a new one
     if (!scope.browser) {
-      scope.browser = await scope.driver.launch({ headless: !session_vars.get_debug() });
+      scope.browser = await scope.driver.launch({ headless: !session_vars.get_debug(), devtools: session_vars.get_debug() });
```

Comment:
Nice find!

Reply:
Just under that I see a link to https://github.com/puppeteer/puppeteer#debugging-tips with more tips.